### PR TITLE
Refactor: Migrate login logo to SVG

### PR DESCRIPTION
**Pull request by SaltyAom**
The current `Login Logo` takes a noticeably long time to load on an actual device.
This happens because as the image size grows large, the file size would be large too, currently, it's a 7K image which is a very high-resolution image. However, because of its high resolution, its size grew big too resulting in a longer download time.

This PR introduces migrating the `login-logo.svg` to `dragon.svg` instead.

[SVG](https://en.wikipedia.org/wiki/Scalable_Vector_Graphics) is a format used for Vector Image which basically tells which line to draw directly, making it has no "real pixel", no matter how much you zoom in, it will always be in a high resolution.
This format is usually used in shape images with low colors like `login-logo.jpg`. Making it perfectly suitable for SVG image.

Migrating to `login-logo` to SVG would reduce to total image size by 35x.
- Former: 391KB
- New: 11.2KB with GZIP (Size without compression is 30KB)

Current: 391KB
jpg

SVG: 11.2KB
svg

So basically, free performance gains and faster page load by 35x.
You might also want to upload this image to CDN like Amazon S3, just like what you did.

### CODE REVIEW

**Comment by AndrewPMurray** [comment]
Thank you for the suggestion! I had considered using SVG for most of my background images, as part of refactoring sections of this app. This gives me a great idea of how to do that not just for this logo but also for the background images on the user home page. I reviewed and will approve this merge.
